### PR TITLE
Streamline cloud build step ID.

### PR DESCRIPTION
**Pull request by xingao267**
#15

### CODE REVIEW

**Comment by xingao267** [comment]
https://console.cloud.google.com/cloud-build/builds/02a50701-05f4-4577-9da4-811175a2ef9f;step=0?project=dpt-demo-3-devops

File: templates/tfengine/components/cicd/manual/configs/tf-apply.yaml [hunk 1]
```diff
@@ -18,45 +18,45 @@ steps:
...
-    id: Display Terraform version
+    id: Terraform version
 
   # live/cicd
   - name: "gcr.io/cloud-foundation-cicd/cft/developer-tools@sha256:a09ad46f513847d31e2cfe20f818c2d64809e7d12695e2e6098900296f6527bf"
     entrypoint: terraform
     args: ["init"]
     dir: "${_TERRAFORM_ROOT}/live/cicd"
-    id: Initialize Terraform for CICD component changes
+    id: Terraform initialization (CICD)
```
Comment (MartinPetkov) [inline templates/tfengine/components/cicd/manual/configs/tf-apply.yaml:28]: Why not just "init"?

Comment (xingao267) [reply templates/tfengine/components/cicd/manual/configs/tf-apply.yaml]: Done

**Comment by xingao267** [comment]
https://console.cloud.google.com/cloud-build/builds/aec2fd58-bc36-4fe5-aaf3-a9c2cdfe6201?project=dpt-demo-3-devops

**Pull request merged**